### ISSUE TITLE
Questions for FAQ

### ISSUE THREAD

**Issue by emjotde** [marian-nmt/marian#133]
Hi,
I am setting up a FAQ page on the website that I will slowly fill with answers. Take a look at the list of questions and add your own questions in this issue. I will try to add them to the page if they are of general interest. 

Here's the mostly empty FAQ: https://marian-nmt.github.io/faq

**Comment by alvations** [comment]
**How do I train a model like in Edinburgh’s WMT2017 submission?**

Is that with `--best-deep`?

**Comment by lparam** [comment]
**Which type of model should i choose?**

**Comment by emjotde** [comment]
@alvations 
Take a look at the new answer. You need to reuse the Transformer examples and use the provided commands in "How do I train a model like in Edinburgh’s WMT2017 submission?" You would need to add your own back-translated data.

**Comment by 1nsunym** [comment]
Is there a translation speed advantage over tensorflow, pytorch or other python ml libraries?
